cmd/gokr-packer/packer.go

```diff
@@ -713,6 +801,30 @@ func logic() error {
 		fromLiteral: *httpsPort,
 	})
 
+	for pkg1, fs1 := range extraFiles {
+		// check against root fs
+		paths := getDuplication(root, fs1)
+		if len(paths) > 0 {
+			return fmt.Errorf("extra files of package %s collides with rootfs: %v", pkg1, paths)
+		}
+
+		// check against other packages
+		for pkg2, fs2 := range extraFiles {
+			if pkg1 == pkg2 {
+				continue
+			}
+			paths = getDuplication(fs1, fs2)
+			if len(paths) > 0 {
+				return fmt.Errorf("extra files of package %s collides with package %s: %v", pkg1, pkg2, paths)
+			}
+		}
+
+		// add extra files to rootfs
+		if err = root.combine(fs1); err != nil {
```

Comment:
if err := root.combine(fs1); err != nil {

(note the :=)

Reply:
changed